model/Cargo.toml

```diff
@@ -11,11 +11,21 @@ edition = "2018"
 [features]
 default = []
 with-diesel = ['diesel']
+sgx = ['secp256k1', 'openssl', 'hex', 'secp256k1/serde']
 
 [dependencies]
 bigdecimal = { version = "0.1.0", features = ["serde"]}
 chrono = { version = "0.4", features = ["serde"]}
-diesel = { version = "1.4", optional = true }
+rand = "0.7.3"
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = "^1.0"
 thiserror = "1.0"
+
+diesel = { version = "1.4", optional = true }
+hex = { version = "0.4", optional = true}
+secp256k1 = { version = "0.17", optional = true }
```

Comment:
@prekucki Same here, any chance we could use `secp256k1` version `0.19`? Otherwise, I'm unable to build `gfaas` since `ya-runtime-wasi` pulls a mismatching `cc` version required by `secp256k1`. Version `0.19` fixed that by changing the dependency requirements.

Reply:
```suggestion
secp256k1 = { version = "0.19", optional = true }
```

Reply:
fixed.